src/content/en/fundamentals/accessibility/01-focus/focus-six.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: shared/narrow
+title: "Focus in Complex Components"
+description: "Managing focus with complex or custom components"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 6
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+Managing focus when you change something on the page is important, but sometimes you need to manage focus at the control level &mdash; for example, if you're building a custom component.
+
+Consider the native `select` element. It can receive basic focus but, once there, you can use the arrow keys to expose additional functionality (the selectable options). If you were building a custom select element, you would want to expose these same kinds of behaviors so that users who rely primarily on the keyboard could still interact with your control.
```

Comment:
Good place for an image https://drive.google.com/open?id=0BxoQ5N_Uc9G-MXBkS2F2dGZiNzA